404.html

```diff
@@ -19,25 +19,50 @@
 			['cl', '/es/'],
 			['co', '/es/'],
 			['cr', '/es/'],
+			['da-DK', '/dk/'],
+			['de-DE','/de/'],
 			['ec', '/es/'],
+			['es-ES', '/es/'],
+			['fi-FI', '/fi/'],
+			['fr-FR', '/fr/'],
 			['gt', '/es/'],
 			['hk_zh', '/tw/'],
+			['it-IT', '/it/'],
+			['ja-JP', '/jp/'],
+			['ko-KR', '/kr/'],
 			['la', '/es/'],
 			['lu_de', '/de/'],
 			['lu_fr', '/fr/'],
 			['mx', '/es/'],
+			['nb-NO', '/no/'],
+			['nl-NL', '/nl/'],
 			['pe', '/es/'],
 			['pr', '/es/'],
 			['pt', '/br/'],	
+			['pt-BR', '/br/'],
+			['sv-SE', '/se/'],
+			['zh-Hans-CN', '/cn/' ],
+			['zh-Hant-TW', '/tw/'],
 		]	
 	);
 	
 	(function (win, doc) {
+		const eduRedirectPath = 'education/express';
+		const eduPageRequest = '/express/edu/?locale=';
 		const { href, pathname } = win.location;	
-		const [_ , userLocale] = pathname.split('/');
-		const redirectMapping = redirectMap.get(userLocale) || '/';  
-		const redirectURL = href.replace(`/${userLocale}/`, redirectMapping);
+		let userLocale, redirectURL, redirectMapping;
 
+		if  (pathname.includes(eduPageRequest)) {
+			[_, userLocale] = pathname.split('=');
```

Comment:
Does this really work? I'd expect `?locale=` to be in `location.search`, not `location.pathname?

Reply:
@rofe You are indeed correct. My local test case did not catch that. Since I had hard-coded some URLs for the test, 
it worked fine. On a live instance, it would have definitely failed. Thanks.  

By the way, how do I test this, so that the local 404.html gets triggered? the only way that I am able to test this is by requesting directly http://localhost:3000/404.html; otherwise, it appears that the redirect is being done  at the CDN - not using my local 404.html. For instance, if use http://localhost:3000/express/edu/?locale=cy-GB , it gets directed redirected https://www.adobe.com/tw/education/express/?locale=cy-GB.  How do I get around this?

Reply:
You cannot actually go around this and need the changes to be deployed to a branch.